storybook/storybook-react/src/Dialog/Dialog.stories.tsx

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Button, Dialog, Heading, Paragraph } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Containers/Dialog',
+  component: Dialog,
+  args: {
+    title: 'Later verder gaan',
+    children: [
+      <Paragraph key={0}>
+        U kunt de ingevulde antwoorden opslaan in onze beveiligde database, deze kunt u later openen om verder te gaan
+        met invullen. Nadat u op opslaan heeft geklikt ontvangt u een mail. Met de link in deze mail kunt verder gaan
+        met het formulier.
+      </Paragraph>,
+    ],
+    actions: [
+      <>
+        <Button
+          key={1}
+          type="button"
+          variant="tertiary"
+          autoFocus
+          onClick={() => document.querySelector('dialog')?.close()}
+        >
+          Terug
+        </Button>
+        <Button key={2} type="submit">
+          Verder
+        </Button>
+      </>,
+    ],
+  },
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+  },
+} satisfies Meta<typeof Dialog>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    open: true,
+  },
+  render: (args) => (
+    <div style={{ minHeight: '500px' }}>
+      <Dialog {...args} />
+    </div>
+  ),
+}
+
+export const NoActions: Story = {
```

Comment:
Do we want to keep this story? I thought we decided that actions are required for Dialogs?